main.go

```diff
@@ -129,8 +129,8 @@ func checkinBook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := checkout(book, co.Who); err != nil {
-		log.Println("could not check out book: [%s]", err.Error)
+	if err := checkin(book, co.Review); err != nil {
+		log.Println(fmt.Sprintf("could not check out book: [%s]", err.Error()))
```

Comment:
Consider changing the log line message..... "unable to check in book"?

Reply:
If you want to be more familiar with logging done in the ONSdigitial services, you can import the `github.com/ONSdigital/log.go/log` and use `log.Event`. But the above looks nice. Nicely done Cat

Reply:
Thank you! 
I'll keep it in mind for the next PRs